.travis.yml

```diff
@@ -14,6 +14,8 @@ branches:
 _integration_test: &integration_test
   os: linux
   language: minimal
+  before_install:
+    - git diff --name-only "$TRAVIS_BRANCH"...HEAD | grep -v '^docs/' || travis_terminate 0
```

Comment:
Nit: you can drop the ending `HEAD`: it's implied if not included.  
It's worth a comment that this only works for pull-request branches which is the normal development case for Skaffold since we all work from forks.

Reply:
done